OracleUnifiedDirectory/imagetool/12.2.1.4.0/README.md

```diff
@@ -1,272 +1,10 @@
 Building an Oracle Unified Directory image with WebLogic Image Tool
 ===================================================================
 
-## Contents
+# Introduction
 
-1. [Introduction](#1-introduction)
-2. [Prerequisites](#2-prerequisites)
-3. [Setup WebLogic Image Tool](#3-setup-weblogic-image-tool)
-4. [Download the required packages/installers](#4-download-the-required-packagesinstallers-patches)
-5. [Required build files](#5-required-build-files)
-6. [Additional build commands](#6-additional-build-commands)
-7. [Steps to Create Oracle Unified Directory image](#7-steps-to-create-oracle-unified-directory-image)
-8. [Generate Sample dockerfile with imagetool](#8-generate-sample-dockerfile-with-imagetool)
+For instructions on building an OUD image with WebLogic Image Tool see [Create or update image](https://oracle.github.io/fmw-kubernetes/oud/create-or-update-image#create-or-update-an-oracle-unified-directory-image-using-the-weblogic-image-tool).
```

Comment:
```suggestion
Detailed instructions on how to [create or update an Oracle Unified Directory container image](https://oracle.github.io/fmw-kubernetes/oud/create-or-update-image#create-or-update-an-oracle-unified-directory-image-using-the-weblogic-image-tool) using the WebLogic Image Tool can be found in the [Oracle Fusion Middleware on Kubernetes](https://oracle.github.io/fmw-kubernetes/) documentation.
```

However, the URL to create-or-update-image does not exist and there does not appear to be any open PR that would create it.